src/v1/providers/storage.ts

```diff
@@ -97,13 +97,13 @@ export class BucketBuilder {
 /**
  * The Google Cloud Storage object builder interface.
  *
- * Access via [`functions.storage.object()`](providers_storage_.html#object).
+ * Access via `functions.storage.object()`.
  */
 export class ObjectBuilder {
-  /** @hidden */
+  /** @internal */
   constructor(private triggerResource: () => string, private options: DeploymentOptions) {}
 
-  /** @hidden */
+  /** @internal */
   onChange(): Error {
```

Comment:
Can we remove this in launch.next?

Reply:
👍 https://github.com/firebase/firebase-functions/pull/1220